README.md

```diff
@@ -1,2 +1,21 @@
-# hof-countries
-Typeahead countries
+# Typeahead Countries
```

Comment:
Maybe `Home Office Country List` this could in theory be used outside of a country list it's just a list of countries...